src/sentry/issues/endpoints/group_similar_issues_embeddings.py

```diff
@@ -82,9 +83,12 @@ def get(self, request: Request, group) -> Response:
         stacktrace_string = ""
         if latest_event and event_content_has_stacktrace(latest_event):
             grouping_info = get_grouping_info(None, project=group.project, event=latest_event)
-            stacktrace_string = get_stacktrace_string(grouping_info)
+            try:
+                stacktrace_string = get_stacktrace_string(grouping_info)
+            except TooManyOnlySystemFramesException:
+                stacktrace_string = ""
 
-        if stacktrace_string == "" or not latest_event:
+        if not stacktrace_string or not latest_event:
```

Comment:
Just because I always forget how Python behaves I test the following:
```
>>> def foo(b):
...   if not b:
...     print("hey")
... 
>>> foo("")
hey
>>> foo(None)
hey
>>> foo("bar")
>>> 
```